qubes/tests/api_admin.py

```diff
@@ -1053,6 +1053,32 @@ def test_160_pool_add(self, mock_parameters, mock_drivers):
         )
         self.assertTrue(self.app.save.called)
 
+    @unittest.mock.patch("qubes.storage.pool_drivers")
+    @unittest.mock.patch("qubes.storage.driver_parameters")
+    @unittest.expectedFailure
```

Comment:
It's better to wrap the line you expect to fail with `with self.assertRaises(qubes.exc.QubesException):` (or other exception you want) instead of marking as expected failure. It will be more precise how/what fails this way.